build-release-tools/version.txt

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Is this file needed anymore?  Looks like latest commit removed the code that was using this.